library/std/src/sys/pal/unsupported/time.rs

```diff
@@ -27,6 +27,10 @@ impl Instant {
 }
 
 impl SystemTime {
+    pub const MAX: SystemTime = Duration::ZERO;
+
+    pub const MIN: SystemTime = Duration::MAX;
```

Comment:
I believe these two are mixed up?

Reply:
fixed in 348d68443cfaa067d5235a8d7606214cc56bb999